### ISSUE TITLE
Failure in Facebook

### ISSUE THREAD

**Issue by Kasher** [rg3/youtube-dl#8035]
Hi,
I ran into this video:
https://www.facebook.com/Lotto.il/videos/1188488004498004/?comment_tracking={%22tn%22%3A%22O%22}

Which is a public video, so youtube-dl should be able to download it (even without username and password parameters). Despite that, when I try to download the video, I get the following error message:

`./youtube-dl "https://www.facebook.com/Lotto.il/videos/1188488004498004/?comment_tracking={%22tn%22%3A%22O%22}"`
`[facebook] 1188488004498004: Downloading webpage`
`ERROR: The video is not available, Facebook said: "The link you followed may have expired, or the page may only be visible to an audience you&#039;re not in."`

Basically, what I would want is to get a direct link to the stream, one which I can use as an input to a media player to watch the video.

Thanks in advance!

**Comment by dstftw** [comment]
It does not work without logging in for me.

**Comment by dstftw** [comment]
So you have to either provide account credentials with `--username` and `--password` or pass authenticated cookies [with `--cookies`](https://github.com/rg3/youtube-dl#how-do-i-pass-cookies-to-youtube-dl). Note that first option is broken in current version.
